www/include/configuration/configNagios/formNagios.php

```diff
@@ -54,11 +54,12 @@
 
 $defaultEventBrokerOptions['event_broker_options'][-1] = 1;
 
-if (($o == "c" || $o == "w") && $nagios_id) {
-    $dbResult = $pearDB->query("SELECT * FROM cfg_nagios WHERE nagios_id = '" . $nagios_id . "' LIMIT 1");
+if (($o == "c" || $o == "w") && $nagiosId) {
```

Comment:
Maybe we could use strict comparison on the object ? (even if you did not touched this code :) )
```suggestion
if (($o === 'c' || $o === 'w') && $nagiosId) {
```